### ISSUE TITLE
breakpointsViewer

### ISSUE THREAD

**Issue by michaelkonecny** [GavriYashar/Matlab-Editor-Plugin#59]
This is a feature request.

Analogically to bookmarksViewer, it would be nice to be able to view all breakpoints in a window.

One can get them now by typing `dbstatus` in the console, but that's really not very nice or easy to use.

Here's an example what it looks like in Visual Studio, where I find it really useful:

![breakpoints](https://cloud.githubusercontent.com/assets/4976024/23560653/80e0f156-003b-11e7-96d0-77d237b26d2e.PNG)

**Comment by GavriYashar** [comment]
this is a good idea, but i'll try to mimic intellij fort this matter since i'm using it.

**Comment by michaelkonecny** [comment]
> (edit GY) Nice to have: dockable window
Yes. Dockable windows would be very nice to have overall, even for file structure, bookmark viewer and clipboard stack.

**Comment by GavriYashar** [comment]
- `com.mathworks.mde.editor.breakpoints.MatlabDebugger` in mde.jar

**Comment by GavriYashar** [comment]
`d = com.mathworks.mde.editor.breakpoints.MatlabBreakpointUtils.getDebugger()`
`d.getBreakpoints()`

**Comment by GavriYashar** [comment]
- [ ] display breakpoints
  - [ ] display line breakpoints
  - [ ] display anonymous breakpoint
  - [ ] display conditional break point
- [ ] display stop if error
  - [ ] always
  - [ ] message id
- [ ] display stop if warning
  - [ ] always
  - [ ] message id
- [ ] display stop if NaN or inf
